test/ui-pytest/driver/aut.py

```diff
@@ -67,10 +63,6 @@ def launch(self, *args) -> 'AUT':
                           f'"{self.path}"'
                       ] + list(args)
             local_system.execute(command)
-            try:
-                local_system.wait_for_started(self.process_name)
```

Comment:
Is the wait no longer necessary?

Reply:
well, we have a PID after executing command, that means process started, I think it was extra action

Reply:
Fair.